com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -993,7 +992,7 @@ static void InitializeAdditionalCameraData(Camera camera, UniversalAdditionalCam
         }
 
         static void InitializeRenderingData(UniversalRenderPipelineAsset settings, ref CameraData cameraData, ref CullingResults cullResults,
-            bool anyPostProcessingEnabled, out RenderingData renderingData)
+            bool anyPostProcessingEnabled, out RenderingData renderingData, CommandBuffer cmd)
```

Comment:
nitpick: rearrange the parameters order so that the out parameter (renderingData) is last?

Reply:
fixed in 3939959